### PR TITLE
[FIX]l10n_cn: account 2241 is missing in small business template.

### PR DESCRIPTION
**Pull request by cysnake4713**
Based on http://kjs.mof.gov.cn/zhengwuxinxi/zhengcefabu/201111/P020111118325852734144.pdf 
(China Accounting Standards for Small-sized Enterprises; 2011),
and http://www.casplus.com/rules/accounting/new%20cas/Appendix.pdf 
(China Accounting Standards for Busness Enterprises; 2006).
Both must have account 2241, but small one missed it, now add it back.

### CODE REVIEW

**Comment by cysnake4713** [comment]
@OpenERPJeff can you confirm this?

last pr forget cla

**Comment by cysnake4713** [comment]
@nim-odoo recently I'm working on l10n_cn model, Except this bug, I have a lot of other [FIX] and [UPD] PR for l10n_cn model, should I make each commit in one PR or create PR for each commit? Since Every commit is about Improve l10n_cn module, Can I just create one PR?

**Comment by nim-odoo** [comment]
@cysnake4713 : if all your fixes are related to the same l10n_cn model, it is ok to make one PR. Just keep in mind that we will accept only fixes in stable version, no improvement. If improvement need to be made, please do it in the master branch. We will also keep minimum modifications to the accounting since this module has been rewritten for the upcoming version.

About this PR, it is fine to submit a single commit with an explanation of all your changes. However, I would appreciate the confirmation of a Chinese partner or integrator.

Regards

**Comment by nim-odoo** [comment]
Without any feedback from @OpenERPJeff , who is the original author of the file, I will have to close this PR.

**Comment by OpenERPJeff** [comment]
Hi nim, 

I confirm this is a good fix for existing bug in l10n_cn

Hi Matt,

Geat Job, please keep going.

File: addons/l10n_cn/account_chart_small_business_template.xml [hunk 1]
```diff
@@ -252,7 +252,7 @@ http://kjs.mof.gov.cn/zhengwuxinxi/zhengcefabu/201111/t20111107_605525.html
             <field name="user_type" ref="user_type_bs39"/>
         </record>
 
-        <record id="small_business_chart1602" model="account.account.template">
+        <record id="small_business_chart1622" model="account.account.template">
```
Comment (nim-odoo) [inline addons/l10n_cn/account_chart_small_business_template.xml]: Although you are right, we cannot change an ID.

Comment (cysnake4713) [inline addons/l10n_cn/account_chart_small_business_template.xml]: If we don't change this one, there will be two 'small_business_chart1602' in this file, the first will be override by this one, If don't change this, Maybe I will change the first one?

Comment (nim-odoo) [inline addons/l10n_cn/account_chart_small_business_template.xml]: ok, my bad, I didn't notice that. This is fine then.

**Comment by cysnake4713** [comment]
@nim-odoo After these last two commits this pr can be fully merged. Already Discussed with Jeff.
@OpenERPJeff  please confirm the last two commit, 

635324d is highly recommended to be merged, Another plugin 'oecn_account_print' is widely used in china, We already modify 'oecn_account_print' to fit in 635324d, The new version make chinese user in print legal style  'balance sheet' and 'Profit and Loss Statement'. Without this commit we have to maintain two different versions of l10n_cn. Since  'balance sheet' and 'Profit and Loss Statement' is the most important Financial Statements, Miss Those two make odoo hard to spread in China.

22d33c5 maybe inappropriate in this pr, However this is make account view much more cooler to display, And I think merging it may cost no compatibility problem. If this commit can't be merge here, I will pull it in master branch.

**Comment by nim-odoo** [comment]
@cysnake4713 : please stop adding commits in this PR. I merged all of them in this other PR: https://github.com/odoo/odoo/pull/7976 If you wish, you can provide me a full description of your modifications.

**Comment by cysnake4713** [comment]
@nim-odoo what should I do about The last new commit 635324d 22d33c5 ? Maybe create another PR?

**Comment by nim-odoo** [comment]
That's ok for now, I will merge everything at once.

**Comment by jeffery9** [comment]
great job. thanks

**Pull request closed without merging**